### DISCUSSION TITLE
Filtering duplicate articles across multiple feeds

### DISCUSSION THREAD

**Discussion by LostLaplace** [FreshRSS/FreshRSS#6535] [Q&A]
Hey all, 

I primarily use FreshRSS to aggregate RSS feeds from news publications that I read through. Namely, NYT, WSJ, and WaPo. Each of these organizations post RSS feeds that sometimes overlap with other feeds within the same organization. For example, here is the list of NYT RSS feeds: https://www.nytimes.com/rss

As an example, [this ](https://www.nytimes.com/2024/06/03/us/politics/sheila-jackson-lee-pancreatic-cancer.html) article was recently posted to both NYT->US->Politics and NYT->US News. I am interested in articles in both of these categories (the politics feed is not a perfect subset of the US News feed, only occasional articles are posted to both), but I am annoyed that repeat articles (with the same exact URL) show up twice in my aggregator.

Is there any mechanism to avoid duplicate entries like this from separate feeds?

**Comment by LostLaplace** [comment]
I saw a recommendation in another thread to try making the "URL" field of the database unique. I find the SQLite DB to be much easier to use, but I haven't found a good way to alter the schema of this DB, even diving into the files in my installation.

**Comment by Alkarex** [comment]
There is not yet a mechanism for that, but there are plans to add some more tools at category and global levels (we might have some existing issues already - linking welcome).
For now, a way to mitigate the issue is to use the existing feature to automatically mark as read articles if you can find an indication that they will appear in another feed. For instance, mark as read articles appearing in *NYT->US News* if they have *Politics* as tag or in the URL somehow.

**Comment by giuseppe998e** [comment]
Still no fix for that?

**Comment by Alkarex** [comment]
Since FreshRSS 1.25.0, there is an option to automatically mark as read articles with identical title within the same category https://github.com/FreshRSS/FreshRSS/pull/6922
